CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [1.14.1]
```

Comment:
It should be major version, since it's `feat:`. Releaser follows angular commits style for semver